code/game/objects/items/toys.dm

```diff
@@ -768,3 +768,95 @@
 	. = ..()
 	if (proximity)
 		visible_message("<span class='warning'>\The [src] says, \"[pick(possible_answers) ]\" as it hits \the [O]!</span>")
+
+
+//////////////////////////////////////////////////////
+//					Chess Pieces					//
+//////////////////////////////////////////////////////
+
+/obj/item/toy/chess
+	name = "chess piece"
+	desc = "This should never display."
+	icon = 'icons/obj/items/chess.dmi'
+	w_class = ITEM_SIZE_SMALL
```

Comment:
I don't actually know how obvious it is that these are big and the others are small; there are lots of not-to-scale sprites. I've never seen the others in game so could be off the mark, but would argue that either (a) replace the others or (b) make it more transparent that these are big via item var values and desc.

Reply:
Isn't there a long-term goal of making world sprites to-scale? That's why the icon system is the way it is.
That said, updating the w_class value might be good, yes.